betelgeuse.py

```diff
@@ -110,10 +143,171 @@ def parse_test_results(test_results):
     return Counter([test['status'] for test in test_results])
 
 
+def add_test_case(args):
+    """Task that create or update test cases and manage manage its requirement.
```

Comment:
I will stick with requirement since it is just one requirement that is created/fetched.